blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
 image: 
 category: 3
 summary: |
-  This blueprint will help you use the Genesys Bot Connector to connect to a Microsoft Power Virtual Agent to answer customer queries via Web Messaging. It also provides a guide to do this for any other bot technology that Genesys does not support as a strategic vendor.
+  This Genesys Cloud Developer Blueprint explains how to deploy a Microsoft Power Virtual Agent (VA) bot in Genesys Cloud. The blueprint solution uses Genesys Bot Connector that provides the API to connect to the bot. The Genesys Bot Connector acts as the link between Genesys Cloud and the bot.
 ---
```

Comment:
Web Messaging is our newest messaging channel that replaces our outdated chat capabilities. We want to showcase this capability with this blueprint. How about:

This Genesys Cloud Developer Blueprint explains how to use the Genesys Bot Connector to connect to a Microsoft Power Virtual Agent to answer customer queries via Web Messaging.

Reply:
Looks like the first "T" is missing (line 10).
Also, web messaging is lowercase; Messenger is uppercase.